.vscode/settings.json

```diff
@@ -15,5 +15,10 @@
     },
     "[typescript]": {
         "editor.defaultFormatter": "esbenp.prettier-vscode"
-    },
-}
\ No newline at end of file
+	},

```

Comment:
why do we need that? the prettier plugin should take the configuration  in the repo

Reply:
Using the prettier extension on VSCode does not use the prettier config file.

Reply:
i'll check that gladly, but it doesn't belong in this PR.

Reply:
Sounds good. I'll open another PR for it.

Reply:
You should revert the change on this one :-)